tests/helm/templates/egress-rule-wildcard-httpbin.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: config.istio.io/v1alpha2
+kind: EgressRule
+metadata:
+  name: httpbin-wildcard
+spec:
+  destination:
+      service: "*.httpbin.org"
+  ports:
+      - port: 80
+        protocol: http
```

Comment:
why do you have egress rules here? It looks like usual e2e tests..

Reply:
The template is a copy (converted to helm syntax) of the pilot integration test config combined
with the e2e simple test config.

The goal is to setup a cluster with all the test apps we need, and run various test suites
against the same baseline. Many of the tests are already idempotent and can be parallel, 
and could save a lot of 'sleep' and wait if they were run against a pre-setup cluster.
Also we have too much inconsistency on how we setup the tests ( pilot's own template 
system and infra, e2e has a different style, etc).